webui/react/src/pages/JobQueue/JobQueue.table.tsx

```diff
@@ -83,8 +85,20 @@ export const columns: ColumnType<Job>[] = [
   {
     key: 'name',
     render: (_: unknown, record: Job): ReactNode => {
-      const label = record.name ?
-        record.name : <span>{jobTypeLabel(record.type)} {truncate(record.entityId, 6, '')}</span>;
+      let label: ReactNode = null;
+      switch (record.type) {
+        case JobType.EXPERIMENT:
+          label = <div>{record.name}
+            <Tooltip title="Experiment ID">
+              {` (${record.entityId})`}
```

Comment:
**suggestion:** the `(#)` makes me think of count or some number. How about prefixing it as a `BadgeTag` component (in front of the experiment name)?

Reply:
![2021-12-06_10-32-41](https://user-images.githubusercontent.com/12127420/144902253-bcdb1838-8e66-4e15-88c9-0e0020502829.png)

I gave it a try I think it grabs too much focus. maybe with some adjustment for an outline style? I'll leave it out of this pr. eventually, we don't want to include the id in the name and bring the id column back.